src/app/modules/MainScreen/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { styled } from '../../styles/themes/defaultTheme';
```

Comment:
dont use relative imports. Please use something like @styles.themes. To achive that there is a need for babel with proper plugin for aliases,.